.github/workflows/deploy.yml

```diff
@@ -0,0 +1,117 @@
+name: Deploy 
+
+on: 
+  push: 
+    branches: 
+      - master 
+  pull_request:
+    branches:  
+      - '**'
+
+env: 
+  EASI_APP_NODE_VERSION: '16.14.0'
+  EASI_APP_GO_VERSION: '1.16.6'
+  IMAGE_TAG: ${{ github.event.pull_request.head.sha || github.sha }}
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  DOCKER_BUILDKIT: 1
+ 
+
+jobs:
+  Build: 
+    uses: ./.github/workflows/initial_build.yml 
+
+  Server_Test: 
+    uses: ./.github/workflows/server_test.yml 
+    needs: Build
+
+  deploy: 
+    strategy:
+      matrix: 
+        env: [dev, impl, prod]
+        deployment_version: [9, 8, 7]
+        exclude: 
+          - env: dev 
+            deployment_version: 8 
+          - env: dev 
+            deployment_version: 7 
+          - env: impl 
+            deployment_version: 9 
+          - env: impl
+            deployment_version: 7 
+          - env: prod
+            deployment_version: 8 
+          - env: prod 
+            deployment_version: 9 
```

Comment:
Instead of this `exclude` block in the matrix definition, maybe it'd be cleaner to calculate which version of the lambda to use with a step inside the job (and you can "output" the right version from the script using the [GH Actions command ::set-output](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter))

Reply:
Or, potentially even better - we could maybe even just use `include:` to define each mapping, rather than `exclude`

Reply:
https://github.com/CMSgov/easi-app/pull/1537/files#r828021602

Closing in favor of this convo